### DISCUSSION TITLE
Tab sends L_Shift on long press even on tap layer with no dual mode keys, no matter where I put the Tab button in Via

### DISCUSSION THREAD

**Discussion by aleksify** [manna-harbour/miryoku#332] [Q&A] [answered]
I use the tap layer for gaming and for lots of games it's important to be able to long press Tab (to see a list of players, or current mission, etc.) But for some reason Tab sends Shift on a long press even though in Via it clearly shows it's just a tab. I also tried to put Tab on different buttons, the problem is still there.

**Comment by manna-harbour** [comment, marked as answer]
See https://github.com/manna-harbour/miryoku/discussions/242#discussioncomment-7380934.
